carapace-server/src/main/java/org/carapaceproxy/server/RuntimeServerConfiguration.java

```diff
@@ -285,18 +286,24 @@ public void configure(ConfigurationStore properties) throws ConfigurationNotVali
 
     private void tryConfigureCertificate(int i, ConfigurationStore properties) throws ConfigurationNotValidException {
         String prefix = "certificate." + i + ".";
-
-        String certificateHostname = properties.getProperty(prefix + "hostname", "");
```

Comment:
nit: it is not a good practice to rename variable if not needed, you are making the diff harder to understand